www/docs/en/dev/guide/platforms/ios/webview.md

```diff
@@ -61,6 +61,8 @@ After using either of these two methods, continue with the **"Using CDVViewContr
 
 1. Add `Carthage/Build/iOS/Cordova.framework` into your Xcode project.
 
+1. Add Cordova.framework into Embedded Binaries in your Xcode General Setting page.
```

Comment:
This section should be added or app will get runtime error which is 'App can not load Cordova dynamic framework.'